plugins/e57/io/E57Writer.cpp

```diff
@@ -288,23 +303,23 @@ void E57Writer::setupWriter()
         {
             auto bounds = e57plugin::getPdalBounds(pdal::Dimension::Id::Red);
             proto.set(e57Dimension,
-                e57::IntegerNode(*m_imageFile, 0, bounds.first,bounds.second));
+                      e57::IntegerNode(*m_imageFile, 0, bounds.first,bounds.second));
         }
         else if (e57Dimension.find("intensity") != std::string::npos)
         {
             auto bounds = pdal::e57plugin::getPdalBounds(
-                pdal::Dimension::Id::Intensity);
+                              pdal::Dimension::Id::Intensity);
```

Comment:
These should be using the intensity bounds you've found while writing.

Reply:
same as above.